stack.go

```diff
@@ -83,19 +78,31 @@ var ErrNoFunc = errors.New("no call stack information")
 //    %+v   equivalent to %+s:%d
 //    %#v   equivalent to %#s:%d
 func (c Call) Format(s fmt.State, verb rune) {
-	if c.fn == nil {
+	frames := runtime.CallersFrames(c.pcs[:])
+	head , _ := frames.Next()
+	frame, _ := frames.Next()
+
+	if head.Function == "runtime.signpanic" {
+		frame, _ = frames.Next()
+	}
```

Comment:
This `if` statement seems unnecessary. I don't want this to skip sigpanic frames.